src/main/java/com/photolocos/enterprise/dao/PhotoDAOStub.java

```diff
@@ -0,0 +1,89 @@
+package com.photolocos.enterprise.dao;
+
+import com.photolocos.enterprise.dto.Photo;
+import org.springframework.stereotype.Component;
+
+import java.util.*;
+
+/**
+ * @author Paramjyot Sandhu (sandhups@mail.uc.edu)
+ *
+ * PhotoDAO class to handle Photo related operations
+ */
+@Component
+public class PhotoDAOStub implements IPhotoDAO {
+
+    Map<Integer, Photo> photos = new HashMap<>();
+
+    @Override
+    public boolean createEntry(Photo photo) throws Exception {
+        photos.put(photo.getPhotoId(), photo);
+        if (photos.containsKey(photo.getPhotoId())) return true;
```

Comment:
The if test here is redundant.  Just make it:

return photos.containsKey(photo.getPhotoId())

That will give you the same result with less typing.